tmt/utils.py

```diff
@@ -3920,7 +3919,7 @@ def format_value(
 
 def format(
         key: str,
-        value: Union[None, int, float, bool, str, list[Any], dict[Any, Any]] = None,
+        value: Union[None, float, bool, str, list[Any], dict[Any, Any]] = None,
```

Comment:
Why is this change? Is it because `int` can be treated like `float`?

Reply:
Yes, this rule to be specific: https://docs.astral.sh/ruff/rules/redundant-numeric-union/

Reply:
But... but... why stopping there, `True` and `False` are also integers :)

Reply:
> Lol, writing C code? :D

```
$ python
Python 3.11.9 (main, Apr 17 2024, 00:00:00) [GCC 13.2.1 20240316 (Red Hat 13.2.1-7)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> type(True)
<class 'bool'>
>>> isinstance(True, int)
True
>>> 
```

> I think if it's to indicate that something is JSON-like stuff, it would be good to be more explicit about that, but I think list[Any] and dict[Any, Any] indicate that it isn't?

Not necessarily JSON, just values. I didn't want to use `Any`, as `format_value()` certainly cannot format everything, just primitives and dicts and lists. It would be better if even these `list` and `dict` wouldn't use `Any`, but also `Union[None, float, bool, str]`, but I guess that's for another patch